src/lib/components/pane/code/editor.svelte

```diff
@@ -140,6 +141,19 @@
             }),
             parent,
         });
+        zoomSize.subscribe((size) => {
```

Comment:
pretty sure this subscription never gets removed, so we'd be leaking resources; better to just use a reactive block before the `onMount`:

(note: this also incorporates other suggestions)
```ts
$: {
    if (view) {
        view.contentDOM.style.fontSize = `${$editorTextSize}px`;
        for (const num of view.scrollDOM.querySelectorAll(".cm-gutterElement")) {
            (num as HTMLElement).style.fontSize = view.contentDOM.style.fontSize;
        }
    }
}
```